README.md

```diff
@@ -38,23 +38,22 @@
 
 ## OpenTelemetry Python
 
-The Python [OpenTelemetry](https://opentelemetry.io/) implementation.
+This page describes the Python [OpenTelemetry](https://opentelemetry.io/) implementation. OpenTelemetry is an observability framework for cloud-native software.
 
 ## Getting started
 
-OpenTelemetry's goal is to provide a single set of APIs to capture distributed traces and metrics from your application and send them to an observability platform. This project allows you to do just that for applications written in Python.
+The goal of OpenTelemetry is to provide a single set of APIs to capture distributed traces and metrics from your application and send them to an observability platform. This project lets you do just that for applications written in Python.
 
 This repository includes multiple installable packages. The `opentelemetry-api`
-package includes abstract classes and no-op implementations that comprise the OpenTelemetry API following
-[the
-specification](https://github.com/open-telemetry/opentelemetry-specification).
+package includes abstract classes and no-op implementations that comprise the OpenTelemetry API following the
+[OpenTelemetry specification](https://github.com/open-telemetry/opentelemetry-specification).
 The `opentelemetry-sdk` package is the reference implementation of the API.
 
-Libraries that produce telemetry data should only depend on `opentelemetry-api`,
-and defer the choice of the SDK to the application developer. Applications may
+Libraries that produce telemetry data only need to depend on `opentelemetry-api`,
```

Comment:
I think the purpose of "should only" is to outline that the sdk should not be taken a dependency on. I think the rewording 'need to" loosens this definition.

Reply:
> The use of "should" and "may" are ambiguous here and should be replaced with more direct language. Are we telling users that the libraries only depend on opentelemetry-api? Or is this somehow a suggestion to the user?

As @codeboten outlined, we are following the specifications for components and behaviour that need to follow "should", "may" and "must" descriptions. They are documented currently to the best of our abilities, but we do not want to change or loosen the definitions.

Reply:
Okay I understand, thanks. I replaced with the original terminology.